### DISCUSSION TITLE
Maxi Size in cache

### DISCUSSION THREAD

**Discussion by yoandypv** [ben-manes/caffeine#490] [Q&A] [answered]
He, pls anybody know what is the cache max size support by 2.6.2 version ??

Thanks

**Comment by ben-manes** [comment, marked as answer]
Roughly ~2^63

https://github.com/ben-manes/caffeine/blob/3ee6531ecb28f4a2c101bf576537d8ac95ff4de5/caffeine/src/main/java/com/github/benmanes/caffeine/cache/BoundedLocalCache.java#L193-L194
